robottelo/hosts.py

```diff
@@ -71,6 +71,18 @@
 }
 
 
+@lru_cache(maxsize=128)
```

Comment:
why the maxsize here?

Reply:
Somehow I was not getting cached value  though the parameter was same for a function and hence I tried applying the maxsize in that case. Now it should not be needed anymore